service/machine.py

```diff
@@ -216,7 +216,7 @@ def upload_privacy_data(machine_request, new_machine):
 
     if is_public:
         print "Marking image %s private" % img.id
```

Comment:
LOL

Reply:
I think this is a matter of the _label_ `'private'` is no longer being used, anything `'private'` is not `'shared'`. The logic here seems confusing on face-value. 

But I think we mark "public" image private during the machine-request phase to avoid them showing up in glance-image catalog.